superset-frontend/plugins/legacy-preset-chart-deckgl/src/utilities/controls.ts

```diff
@@ -39,6 +39,7 @@ export function columnChoices(datasource: Dataset | QueryResponse | null) {
 }
 
 export const PRIMARY_COLOR = { r: 0, g: 122, b: 135, a: 1 };
```

Comment:
@msyavuz @mistercrunch if anyone's keeping a punch list somewhere of themable things, this PRIMARY_COLOR setting could be a good one to match with the theme's primary color.